src/details/ArborX_DetailsMortonCode.hpp

```diff
@@ -114,7 +114,7 @@ unsigned long long morton64(float x, float y)
 {
   // The interval [0,1] is subdivided into 2,147,483,648 bins (in each
   // direction).
-  constexpr unsigned N = (1 << 31);
+  constexpr unsigned N = (1u << 31);
```

Comment:
Per https://github.com/arborx/ArborX/pull/637#discussion_r812279078 plz apply to all

Reply:
You want it just to look the same? There is no real reason to use it elsewhere.